py-polars/polars/io/database/_executor.py

```diff
@@ -348,9 +359,11 @@ def _normalise_cursor(self, conn: Any) -> Cursor:
 
     async def _sqlalchemy_async_execute(self, query: TextClause, **options: Any) -> Any:
         """Execute a query using an async SQLAlchemy connection."""
-        is_session = type(self.cursor).__name__ == "async_sessionmaker"
+        is_session = "session" in type(self.cursor).__name__.lower()
```

Comment:
Can we then also substitute this check with an `isinstance` check?

Reply:
Already done it - and tanother ;)